src/styles/base/variable.scss

```diff
@@ -0,0 +1,37 @@
+//Colors of first page
+$background-color:#F3F2EA;
```

Comment:
Please, rename this color name. 
We don't have a base color of the bg for the project.
Please, name all colors by their name.